kafka-connect-s3/docs/s3_connector.rst

```diff
@@ -95,6 +95,21 @@ NO compatibility, BACKWARD compatibility, FORWARD compatibility and FULL compati
 
 Schema evolution in the S3 connector works in the same way as in the `HDFS connector <../../../connect-hdfs/docs/hdfs_connector.html#schema-evolution>`_.
   
+Automatic Retries
+-----------------
+Users' environments might experience network partitioning, reach AWS throttling limits etc.
+In order to have a robust system it makes sense to have a retry policy, however, it's important
+to avoid a thundering herd problem.
+The S3 connector uses retry policy, predefined by ``com.amazonaws.retry.PredefinedRetryPolicies.SDKDefaultRetryCondition``
+and ``com.amazonaws.retry.PredefinedBackoffStrategies.FullJitterBackoffStrategy`` provided by AWS SDK.
```

Comment:
I liked your suggestion last night of not talking to much about the implementation. I'd remove this.

Reply:
Thanks, removed.

Reply:
✅